docs/installation.md

```diff
@@ -72,34 +72,43 @@ If you want to help develop Lychee, install the development dependencies by remo
 
-#### Copy configuration file
-If you have not copied the `.env.example` file to a new file named `.env`, you should do that now.
+You need to configure some file and directory permissions. Directories within the `storage` and the `bootstrap/cache` directories should be writable by your web server or Lychee will not run. Additionally, `public/uploads`, `public/dist` and `public/sym` need to be writable to upload files, customise the CSS and utilise [symbolic links](https://lycheeorg.github.io/docs/settings.html#symbolic-link) respectively.
+
+#### Application Settings
+
+The main directory should contain a `.env` file.
+Normally, Composer has generated this file for you based on `.env.example` while Composer has been installing the dependencies (see above).
+If the file `.env` does not exist, then you should copy `.env.example` to a new file named `.env` now.
+Note that the `.env` file should **not** be exposed online as it contains the encryption key used for cookies as well as database credentials.
 
-#### Application Key
-The next thing you should do after installing Lychee is set your application key to a random string. This is easily done by using the `php artisan key:generate` command.
+##### Application Key
 
-If you don't want to use this command and set the key yourself, this string should be 32 characters long and can be set in the `.env` environment file.
+The `.env` file contains the setting `APP_KEY` which holds your application key.
+Use the `php artisan key:generate` command to set the application key to a random value.
+If you don't want to use this command and set the key yourself, this value should be 32 characters long.
 
 **If the application key is not set, your user sessions and other encrypted data will not be secure!**
 
-#### Additional Configuration
-Lychee needs almost no other configuration out of the box. You are free to get started developing! However, you may wish to review the `config/app.php` file and its documentation. It contains some options that aren't included from `.env` that you may wish to change according to your installation.
+##### Additional Configuration
 
-You may also want to configure a few additional components of Lychee in `.env`, such as:
+The settings of the default `.env` file provide usable and operational settings out-of-the-box. 
+
+You may want to configure a few additional components of Lychee, such as:
 
 - Cache
 - Database
 - Session
 
+Some advanced options cannot be configured through the `.env` file, but you can take a look into the directory `config/` to find such options. Usually, you don't have to change them and can go with the defaults. **If you don't know what they're doing, do not change them.**
```

Comment:
To me this is more of an invitation than @qwerty287's version. I don't like it.

Reply:
I am not sure if Github is out-of-sync again or if were are looking at different things, but this is @qwerty287's version. (confused)

Reply:
It's different, but less different than I remembered. Did we decide that only the ones in `app.php` were missing from `.env`?

Other than that.... I can't suggest here so...